codeforboston/cliff-effects
# Add linting pre-commit hook via husky and lint-staged packages. Fixes #897

Test plan:

- Change some file in such a way that it fails linting.
- Run `git commit -am 'test commit'`
- The commit should run a lint check first
- Look at the commit diff and make sure it's changed to pass lint
